dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -826,6 +827,144 @@ ShowStatusPropertyDialog(
     }
 }
 
+VOID ShowNetworkIconContextMenu(
+    _In_ HWND hwndOwner,
+    _In_ LANSTATUSUI_CONTEXT *pContext)
+{
+    if (!pContext || !pContext->pNet)
+        return;
+
+    HMENU hMenu = CreatePopupMenu();
+    if (!hMenu) return;
+
+    POINT pt;
+    GetCursorPos(&pt);
+
+    SetForegroundWindow(hwndOwner);
+
+    // The context menu items, set to their default values.
+    struct
+    {
+        UINT uID;
+        UINT uFlags;
+        UINT_PTR uIDNewItem; // wCmdID
+    } MenuItems[] =
+    {
+        {IDS_NET_ACTIVATE, MF_STRING | MF_GRAYED, IDM_NETICON_ENABLE},
+        {IDS_NET_STATUS, MF_STRING | MF_GRAYED, IDM_NETICON_STATUS},
+        {IDS_NET_REPAIR, MF_STRING | MF_GRAYED, IDM_NETICON_REPAIR},
+        {-1, 0, -1}, // Separator
+        {IDS_NET_OPEN_CONNECTIONS, MF_STRING, IDM_NETICON_OPEN_CONNECTIONS},
+        {IDS_NET_PROPERTIES, MF_STRING | MFS_DEFAULT, IDM_NETICON_PROPERTIES},
```

Comment:
```suggestion
        {IDS_NET_OPEN_CONNECTIONS, MF_ENABLED, IDM_NETICON_OPEN_CONNECTIONS},
        {IDS_NET_PROPERTIES, MF_ENABLED | MFS_DEFAULT, IDM_NETICON_PROPERTIES},
```
NOTE: I'm surprised that MFS_DEFAULT is supported as part of the AppendMenu flags, it's not officially documented in MSDN there; it is, however, documented for usage with the MENUITEMINFOW structure and InsertMenuItemW function.